packages/paste-theme/src/types.ts

```diff
@@ -0,0 +1 @@
+export type ValueOf<O> = O[keyof O];
```

Comment:
Should this go in the types package as some kind of utility as we'd got it twice now?